### PR TITLE
refactor(Core/Tools): add braces and impove codestyle

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
## Changes Proposed:
-

### CODE REVIEW

File: src/tools/vmap4_extractor/wmo.cpp [hunk 1]
```diff
@@ -415,17 +421,23 @@ int WMOGroup::ConvertToVMAPGroupWmo(FILE* output, WMORoot* rootWMO, bool precise
     //------LIQU------------------------
     if (LiquEx_size != 0)
     {
-        int LIQU_h[] = { 0x5551494C, static_cast<int>(sizeof(WMOLiquidHeader) + LiquEx_size) + hlq->xtiles * hlq->ytiles }; // "LIQU"
+        int LIQU_h[] = { 0x5551494C, static_cast<int>(sizeof(WMOLiquidHeader) + LiquEx_size) + hlq->xtiles* hlq->ytiles };  // "LIQU"
```
Comment (Kitzunu) [inline src/tools/vmap4_extractor/wmo.cpp:424]: weird change

**Comment by FrancescoBorzi** [comment]
@Winfidonarleyan  what's the status of this PR?

**Comment by Winfidonarleyan** [comment]
> @Winfidonarleyan  what's the status of this PR?

Need remake changes

**Comment by FrancescoBorzi** [comment]
@Winfidonarleyan feel free to reopen this when you want to complete it. Closing for now.

**Pull request closed without merging**